formats/detailed_guide/frontend/examples/detailed_guide.json

```diff
@@ -0,0 +1,71 @@
+    "change_note": "The second entry in the table Examples of salary sacrifice has been amended to correct the explanation of how much of the salary is subject to tax and National Insurance contributions.",
+    "first_published_at": "2014-06-12T10:00:00+01:00"
+  },
+  "links": {
+    "lead_organisations": [
+      {
+        "content_id": "6667cce2-e809-4e21-ae09-cb0bdc1ddda3",
+        "title": "HM Revenue & Customs",
+        "base_path": "/government/organisations/hm-revenue-customs",
+        "api_url": "https://www.gov.uk/api/organisations/hm-revenue-customs",
+        "web_url": "https://www.gov.uk/government/organisations/hm-revenue-customs",
+        "locale": "en",
+        "analytics_identifier": "D25"
+      }
+    ],
+    "worldwide_organisations": [
+      {
+        "content_id": "6667cce2-e809-4e21-ae09-cb0bdc1ddda3",
+        "title": "HM Revenue & Customs",
```

Comment:
Me and @binaryberry can't figure out why we actually put this in, as we don't need it. :flushed: